tests/test_utils.py

```diff
@@ -156,21 +148,17 @@ def __str__(self):
 
     annotation = annotate_source_code(
         TEST_SOURCE_CODE,
-        44,
+        36,
         col_offset=8,
-        context_lines=8,
+        context_lines=4,
         line_numbers=True,
     )
     assert annotation == r"""
-     36
-     37         elif self.lineno is not None and self.col_offset is not None:
-     38             output = 'line %d:%d %s' % (
-     39                 self.lineno,
-     40                 self.col_offset,
-     41                 output
-     42             )
-     43
----> 44         return output
+     32
+     33         elif self.lineno is not None and self.col_offset is not None:
+     34             output = f'line {self.lineno}:{self.col_offset} {output}'
+     35
+---> 36         return output
```

Comment:
I think this test is not really all that great, if we are relying on a particular error string that could change.
@jacqueswww what do you think?

@wschwab: not a comment against this PR, no worries.

Reply:
These tests are just testing that the line marker formatter for error messages is working correctly.  It's not actually a runtime error message.  It's just testing the functionality of a text processing function.